packages/form/src/cli.ts

```diff
@@ -58,14 +125,20 @@ async function create() {
 		});
 	}
 
-	const relativeOutputPath = join(relativeOutputDir, "Form.tsx");
+	const relativeOutputPath = join(
+		relativeOutputDir,
+		toUseFramework.templateDir,
+	);
 	const outputPath = resolve(process.cwd(), relativeOutputPath);
 
-	const reactFormTemplate = await readFile(
-		new URL("../templates/react/Form.tsx", import.meta.url),
-	);
+	const templatePath = resolve(`./templates/${toUseFramework.templateDir}`);
```

Comment:
```suggestion
  const templatePath = fileURLToPath(
    new URL(`../templates/${toUseFramework.templateDir}`, import.meta.url)
  );
```

I noticed this breaks locally since it attempts to resolve from where the cli is called, not from where our internal templates are stored. I think refactoring to use file urls would be a better pick. It's a newer standard that uses import.meta.url to construct a relative path from the current working file.